README.md

```diff
@@ -11,8 +11,8 @@ const $ = require('sanctuary-def');
 ```
 
 The next step is to define an environment. An environment is a list of
-[types](#types) such as `$.Number` and `$.String`. `$.env` is a default
-environment, which may be augmented with custom types:
+[types](#types) such as `$.Number` and `$.String`. [`$.env`](#env) is a
+default environment, which may be augmented with custom types:
```

Comment:
A default environment or the default environment? Are there other possible default environments?